### PR TITLE
Allow for middleware to be inserted

### PR DESCRIPTION
**Pull request by amcgavin**
Add sentry support to the node portion.

### CODE REVIEW

File: render.js [hunk 1]
```diff
@@ -22,6 +24,23 @@ var argv = require('yargs')
...
 
 var app = express();
+
+if (process.env.SENTRY_DSN) {
+  Sentry.init({
+    dsn: process.env.SENTRY_DSN, integrations: [
+      new Sentry.Integrations.Http({tracing: true}),
+      new Tracing.Integrations.Express({app}),
+    ],
+    tracesSampleRate: 0.01
```
Comment (simonwillcock) [inline render.js:34]: Consider making this adjustable via an env var as well?

File: render.js [hunk 2]
```diff
@@ -22,6 +24,23 @@ var argv = require('yargs')
...
 
 var app = express();
+
+if (process.env.SENTRY_DSN) {
+  Sentry.init({
+    dsn: process.env.SENTRY_DSN, integrations: [
+      new Sentry.Integrations.Http({tracing: true}),
+      new Tracing.Integrations.Express({app}),
+    ],
+    tracesSampleRate: Number(process.env.SENTRY_SAMPLE_RATE || '0.01')
```
Comment (jarshwah) [inline render.js:34]: Is `0` an ok value? Need to be able to turn it off

**Comment by jarshwah** [comment]
The reason I'm a bit iffy about this is because this is a public package, and we're now enforcing sentry as a dependency, even if it won't be used by lots of people..

**Comment by amcgavin** [comment]
@jarshwah I agree with your concern.
If we wanted to not include sentry here, we'd have to release a breaking change to allow for the middleware to be inserted earlier.

I'll draft up a change later!

**Comment by mic159** [comment]
@amcgavin pretty sure this is already possible with the current version.

**Pull request closed without merging**